.circleci/config.yml

```diff
@@ -65,6 +65,26 @@ jobs:
        # TODO(PiotrSikora): re-enable once fixed.
        # - run: ci/do_circle_ci.sh check_spelling_pedantic
 
+   macos:
+     macos:
+       xcode: "10.2.1"
+     environment:
+       BAZEL_BUILD_EXTRA_OPTIONS: "--define wasm=v8" # v8 only, WAVM segfaults in tests.
```

Comment:
In release and debug ?

Reply:
Nn `fastbuild`, which is what `ci/mac_ci_steps.sh` runs.

I have separate PR for investigating WAVM builds (#55), so let's not block this one on it.